From eda0c2c41f5ec92dcdecbcf16435744345c2d6fc Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Wed, 4 Feb 2015 19:41:05 -0600
Subject: [PATCH 07/13] remoteproc/pruss: fix typos in couple of traces

Fix the mailbox API name typos in couple of traces.

Signed-off-by: Suman Anna <s-anna@ti.com>
---
 drivers/remoteproc/pruss_remoteproc.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/remoteproc/pruss_remoteproc.c b/drivers/remoteproc/pruss_remoteproc.c
index a865295..f8b415b 100644
--- a/drivers/remoteproc/pruss_remoteproc.c
+++ b/drivers/remoteproc/pruss_remoteproc.c
@@ -604,7 +604,7 @@ static void pru_rproc_kick(struct rproc *rproc, int vq_id)
 	/* send the index of the triggered virtqueue in the mailbox payload */
 	ret = mbox_send_message(pru->mbox, (void *)vq_id);
 	if (ret)
-		dev_err(dev, "omap_mbox_msg_send failed: %d\n", ret);
+		dev_err(dev, "mbox_send_message failed: %d\n", ret);
 }
 
 /* start a PRU core */
@@ -866,7 +866,7 @@ static int pru_rproc_probe(struct platform_device *pdev)
 	pru->mbox = mbox_request_channel(client, 0);
 	if (IS_ERR(pru->mbox)) {
 		ret = PTR_ERR(pru->mbox);
-		dev_err(dev, "omap_mbox_get failed: %d\n", ret);
+		dev_err(dev, "mbox_request_channel failed: %d\n", ret);
 		goto free_rproc;
 	}
 
-- 
1.7.9.5

